### PR TITLE
Fix quick typos

### PR DESCRIPTION
**Pull request by chfoster**
## Pull request overview

Quick typo fixes, addresses the below examples:
image
image

as well as replacing places in the text that still had numbered YAGs rather than spelt out. 

## Pull request checklist

Please check if your PR fulfils the following:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been run locally and are passing (`run_tests_locally()`)
- [x] Code is styled according to tidyverse styling (checked locally with `tidy_code()`)

### CODE REVIEW

File: tests/shinytest/testUI-expected/subjectByIndustry_0.json [hunk 1]
```diff
@@ -4209,7 +4209,7 @@
...
...duation, the industry with the highest proportion of female graduates is <b>Education<\/b>, and the median earnings of females in this industry were <b>£25,600<\/b>. The industry with the highest proportion of male graduates was <b>Professional, scientific and technical activities<\/b> and the median earnings of males in this industry were <b>£34,000<\/b>.<br/><br/> The biggest difference in proportions is seen in <b>Human health and social work activities<\/b> where the proportion of female graduates is <b> 12.6  percentage points higher<\/b> than the proportion of male graduates. The biggest difference in median earnings is seen in <b>Construction<\/b> where the median earnings of male graduates were <b>£10,800  higher<\/b> than the median earnings of female graduates.The group with the highest earnings was  <b>Male<\/b> graduates who worked in the <b>Mining and quarrying<\/b> industry (median earnings of <b>£42,700<\/b>). <b>41.6%<\/b> of female graduates are concentrated in the top 2\n                           industries (either <b>Education<\/b> or <b>Human health and social work activities<\/b>),\n                           whereas for male graduates this is 18%.<br/><br/>",
```
Comment (NR7777) [inline tests/shinytest/testUI-expected/subjectByIndustry_0.json]: Can't quite see how this line has changed from the previous version.

File: tests/shinytest/testUI-expected/subjectByIndustry_1.json [hunk 2]
```diff
@@ -4209,7 +4209,7 @@
...
...duation, the industry with the highest proportion of female graduates is <b>Education<\/b>, and the median earnings of females in this industry were <b>£25,600<\/b>. The industry with the highest proportion of male graduates was <b>Professional, scientific and technical activities<\/b> and the median earnings of males in this industry were <b>£34,000<\/b>.<br/><br/> The biggest difference in proportions is seen in <b>Human health and social work activities<\/b> where the proportion of female graduates is <b> 12.6  percentage points higher<\/b> than the proportion of male graduates. The biggest difference in median earnings is seen in <b>Construction<\/b> where the median earnings of male graduates were <b>£10,800  higher<\/b> than the median earnings of female graduates.The group with the highest earnings was  <b>Male<\/b> graduates who worked in the <b>Mining and quarrying<\/b> industry (median earnings of <b>£42,700<\/b>). <b>41.6%<\/b> of female graduates are concentrated in the top 2\n                           industries (either <b>Education<\/b> or <b>Human health and social work activities<\/b>),\n                           whereas for male graduates this is 18%.<br/><br/>",
```
Comment (NR7777) [inline tests/shinytest/testUI-expected/subjectByIndustry_1.json]: again not sure what has changed.

File: tests/shinytest/testUI-expected/subjectByIndustry_2.json [hunk 3]
```diff
@@ -3381,7 +3381,7 @@
...
...evel 8 graduates of all subjects, 5 years after graduation, the industry with the highest proportion of graduates is the same for both female and male graduates (<b>Education<\/b>). The median\n                                earnings for females in this industry  were <b>£35,800<\/b> and for males were <b>£37,600<\/b>.<br/><br/> The biggest difference in proportions is seen in <b>Human health and social work activities<\/b> where the proportion of female graduates is <b> 9.8  percentage points higher<\/b> than the proportion of male graduates. The biggest difference in median earnings is seen in <b>Financial and insurance activities<\/b> where the median earnings of male graduates were <b>£40,000  higher<\/b> than the median earnings of female graduates.The group with the highest earnings was  <b>Male<\/b> graduates who worked in the <b>Financial and insurance activities<\/b> industry (median earnings of <b>£69,900<\/b>). <b>69.8%<\/b> of female graduates are concentrated in the top 2\n                           industries (either <b>Education<\/b> or <b>Human health and social work activities<\/b>),\n                           whereas for male graduates this is 53.5%.<br/><br/>",
```
Comment (NR7777) [inline tests/shinytest/testUI-expected/subjectByIndustry_2.json]: Not sure what's changed.

File: tests/shinytest/testUI-expected/subjectByIndustry_3.json [hunk 4]
```diff
@@ -1473,7 +1473,7 @@
...
+    "crosstab_text": "For level 8 graduates of Allied health, 5 years after graduation, the industry with the highest proportion of female graduates is <b>Education<\/b>, and the median earnings of females in this industry were <b>£38,000<\/b>. The industry with the highest proportion of male graduates was <b>Human health and social work activities<\/b> and the median earnings of males in this industry were <b>£58,800<\/b>.<br/><br/> The biggest difference in proportions is seen in <b>Education<\/b> where the proportion of female graduates is <b> 13.1  percentage points higher<\/b> than the proportion of male graduates. The biggest difference in median earnings is seen in <b>Human health and social work activities<\/b> where the median earnings of male graduates were <b>£23,000  higher<\/b> than the median earnings of female graduates.The group with the highest earnings was  <b>Male<\/b> graduates who worked in the <b>Human health and social work activities<\/b> industry (median earnings of <b>£58,800<\/b>). <br/><br/>",
```
Comment (NR7777) [inline tests/shinytest/testUI-expected/subjectByIndustry_3.json]: Not sure what's changed.

**Review by NR7777 — COMMENTED** [review]
Where a change has been made they are fine. On some of the json example files, I couldn't see what had changed.

**Comment by KirtiDFE** [comment]
I've reviewed and logged my comments in the QA log

**Comment by chfoster** [comment]
Have updated some more bits from the QA log so ready for a second review @NR7777 @KirtiDFE

**Review by NR7777 — APPROVED** [review]

**Pull request merged**